src/components/Table/Table.scss

```diff
@@ -38,6 +38,8 @@ $block: '.#{variables.$ns}table';
         height: inherit;
 
         text-align: start;
+
+        background: var(--g-color-base-background);
```

Comment:
This component doesn't have specific styles and it would be renamed to `BaseTable` later.
New `Table` component will use that base table and provide it's own gravity-ui based styles.
So let's try to avoid styling this component.
You can move this style to demo-component instead.